src/style/internal/shared_input-select-picker.scss

```diff
@@ -54,6 +54,7 @@ $cropped-label-hack--font-size: 0.875rem; //14px
     -moz-osx-font-smoothing: grayscale;
     -webkit-font-smoothing: antialiased;
 
+    color: $input-text-color;
```

Comment:
I squashed this into the fix for `limel-input-field`, as that's the component that was affected by this change too. They're effectively both "making sure label has our default label color" in limel-input-field 🙂